src/options/options.css

```diff
@@ -0,0 +1,72 @@
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    margin: 0;
+}
+
+.root {
+    font-size: 18px;
+    font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    font-weight: 200;
+    display: flex;
+    flex-direction: row;
+}
+
+.route {
+    margin: 20px 20px 0;
+    flex-grow: 1;
+}
+
+.routeTitle {
+    font-size: 1.2em;
+    font-weight: 100;
+    color: #666;
+    margin: 0 0 10px 0;
+    padding-bottom: 10px;
+    border-bottom: 1px solid #ccc;
+}
+
+.button {
+    background: #eee;
+    color: #666;
+
+    font-size: .8em;
+    font-weight: 400;
+    padding: .6em 1.5em;
+    text-transform: uppercase;
+    cursor: pointer;
+
+    outline: 0;
+    border: 1px solid #999;
+    box-shadow: 0 0 1px 0 rgba(0, 0, 0, 0.1);
+    transition: background-color .1s ease-in, color .1s ease-in;
+
+    &:hover {
+        background: #cdcdcd;
+        color: #333;
+    }
+}
```

Comment:
Some styles repeated from [here](https://github.com/aquibm/webmemex-extension/commits/master). I think soon we will combine them